src/js/burials/config/form.js

```diff
@@ -180,8 +180,8 @@ const formConfig = {
           title: 'Burial information',
           path: 'veteran-information/burial',
           uiSchema: {
-            burialDate: currentOrPastDateUI('Date of burial (includes cremation or interment)'),
             deathDate: currentOrPastDateUI('Date of death'),
+            burialDate: currentOrPastDateUI('Date of burial (includes cremation or interment)'),
```

Comment:
I just figured since `burialDate` really comes after `deathDate` when we're looking at the form, we should order it here too.